.changeset/lovely-dolls-taste.md

```diff
@@ -0,0 +1,5 @@
+---
+"wrangler": minor
+---
+
+fix: kv:namespace create to refer to the namespace name instead of the binding name
```

Comment:
Let's expand this description a bit. Without any context I think the current text is a bit cryptic. Perhaps something like:

```
fix: allow `kv:namespace create` to accept a namespace name that contains characters not allowed in a binding name

This command tries to use the namespace name as the binding. Previously, we would unnecessarily error if this namespace name did not fit the binding name constraints. Now we accept such names and then remove invalid characters when generating the binding name.
```

Reply:
For sure. Thanks. :)